src/index.ts

```diff
@@ -115,7 +118,7 @@ export class StateSource<T> {
   public select<R>(scope: Scope<T, R>): StateSource<R> {
     const get = makeGetter(scope);
     return new StateSource<R>(
-      this.state$.map(get).filter(s => typeof s !== 'undefined'),
+      xs.fromObservable<T>(this.state$).map(get).filter(s => typeof s !== 'undefined'),
```

Comment:
According to https://github.com/staltz/cycle-onionify/issues/26 
This stream passed to constructor where `compose` called on it